ui/mirage/factories/kv-metadatum.js

```diff
@@ -36,7 +36,7 @@ const data = {
     4: {
       created_time: '2023-07-21T03:11:58.095971Z',
       deletion_time: '',
-      destroyed: false,
```

Comment:
Since this is a mirage factory - I think we should avoid changing it as much as possible as int should be reserved for defaults. So in this case, we should keep one state present for each version so this should remain the current **and** not destroyed version. 

I think that in the test specifically checking for both icons should edit the metadata instead